src/components/views/homepage/HomeHeader.tsx

```diff
@@ -30,17 +30,17 @@ const HomeHeader = () => {
 				Get rewarded for giving to for-good projects with zero added
 				fees.
 			</Subtitle>
-			<AllProjectsButton
+			<MatchingPoolButton
 				buttonType='primary'
 				size='large'
-				label='SEE PROJECTS'
-				onClick={() => router.push(Routes.Projects)}
+				label='DONATE NOW'
+				onClick={() => router.push(Links.GIVETH_MATCHING)}
```

Comment:
[this link](https://staging.giveth.io/donate/matching-pool-test-0) for staging is working but [the link](https://giveth.io/donate/matching-pool-test-0) for the production after merge it's not working. You can use another link for production.

Reply:
very strange can you look at my changes in links.ts ?